gnosis/safe-react
# fix: add `set`/`deleteGuard`

## What it solves
Resolves #3639

## How this PR fixes it
The collapsed label and correct transaction info displays have been added for `set`/`deleteGuard` calls. A fallback for any future new `SettingsInfo` has been added that will simply show the `type` as well.

## How to test it
Open `rin:0x6cf158C37354d8da5396fb6d3e965318CCf119c1` on the production CGW and observe the correct labelling of the `setGuard` (in the history) and `deleteGuard` (in the queue). When expanding the transaction, the info should be displayed.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/157483423-f7cfe080-0fda-49c9-8f59-fba7a329d353.png)
![image](https://user-images.githubusercontent.com/20442784/157458027-61d67108-1659-4440-98ad-89a6c0ea1979.png)
